### PR TITLE
feature/30 display cartesian plot

### PR DESCRIPTION
**Pull request by leandertolksdorf**
Please test
- display of coordinate system
- display of wave drawing
- display of y-scale text
- setting y-scale in `Oscilloscope.svelte` -> `let scaleY = ...`

### CODE REVIEW

**Comment by leandertolksdorf** [comment]
test

**Review by nicolaskolbenschlag — APPROVED** [review]

**Review by PhlppKrmr — COMMENTED** [review]
Just fyi: those were my concerns

File: Apps/frontend/src/App.svelte [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+  import Oscilloscope from "./views/Oscilloscope.svelte";
 </script>
 
 <main>
-  <Logo />
-  <HelloWorld />
+  <div>
+    <Logo />
+  </div>
+  <div class="canvas">
```
[the remark this answers is not in the document]
Comment (leandertolksdorf) [reply Apps/frontend/src/App.svelte:11]: A canvas can only have one context (webgl, 2d, ...). And for separation of concerns, I created one 2d canvas for the grid and a webgl context for the wave.

File: Apps/frontend/src/views/Oscilloscope.svelte [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  <div class="stack coordinate-system">
+    <CoordinateSystem
+      {canvasWidth}
+      {canvasHeight}
+      {numIntervalsX}
+      {numIntervalsY}
+      yScale={scaleY}
+    />
+  </div>
+  <div class="stack wave">
```
[the remark this answers is not in the document]
Comment (leandertolksdorf) [reply Apps/frontend/src/views/Oscilloscope.svelte:35]: If your referring to setting the wave color: this is currently done with a parameter passed to the WebGLLine in SineWave.svelte

File: Apps/frontend/src/views/Layout.svelte [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<script>
```
[the remark this answers is not in the document]
Comment (leandertolksdorf) [reply Apps/frontend/src/views/Layout.svelte:1]: Doesn't have a purpose, I simply forgot about it...

File: Apps/frontend/src/views/Oscilloscope.svelte (hunk 2 above)
[the remark this answers is not in the document]
Comment (PhlppKrmr) [reply Apps/frontend/src/views/Oscilloscope.svelte:35]: No, I'm referring to displaying multiple different input signals in the same plot. I don't quite know how the waves will behave when they all receive the same zIndex.

File: Apps/frontend/src/App.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (PhlppKrmr) [reply Apps/frontend/src/App.svelte:11]: I think you misunderstood my question. I just wanted to make sure we don't need the class `canvas` multiple times. Cause if we don't we can simply change it to `id="canvas"` (or whatever id you want to give). I made that recommendation cause AFAIK id's are earier to test.

**Pull request merged**